extensions/amp-story/1.0/amp-story.js

```diff
@@ -137,7 +137,10 @@ const DESKTOP_WIDTH_THRESHOLD = 1024;
 /** @private @const {number} */
 const DESKTOP_HEIGHT_THRESHOLD = 550;
 
-/** @private @const {string} */
+/**
+ * NOTE: If udpated here, update in amp-story-player-impl.js
```

Comment:
You should be able to export the constant from here if you want to

Reply:
I'm getting an `AMP is not defined` when importing the variable. I think because the player is executing without AMP.

It needs to be a `string` in amp-story and a `number` in amp-story-player-impl.
I'm not sure if we could use it even if we could import it 🤔